layouts/shortcodes/dbm-secret.md

```diff
@@ -0,0 +1,3 @@
+Store your password using secret management software such as [Vault](https://www.vaultproject.io/). You can then reference this password as `ENC[<SECRET_NAME>]`: for example, `ENC[datadog_user_database_password]`. See [Secrets Management](/agent/configuration/secrets-management/) for more information.
```

Comment:
It's probably worth being specific about where you can use the `ENC` reference -- I'm guessing that's just in the Agent configuration files?